openhtf/util/multicast.py

```diff
@@ -29,6 +29,10 @@
 
 _LOG = logging.getLogger(__name__)
 
+# The multicast logs can occur frequently, e.g. every couple seconds. Prevent
+# most logs from ending up in the test record by default.
+_LOG.setLevel(logging.WARNING)
+
```

Comment:
This also could have been added into the one-time logs setup in `openhtf.util.logs`. I figured it makes sense for each module to govern its own default logging behavior.